### PR TITLE
Django >= 1.10 Compatibility

### PR DESCRIPTION
**Pull request by guneysus**

### CODE REVIEW

File: restless/dj.py [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import six
 
+from django import get_version as get_django_version
 from django.conf import settings
-from django.conf.urls import patterns, url
+
+if get_django_version() >= '1.10.0':
```
Comment (seocam) [inline restless/dj.py]: This comparison might fail in some circumstances. Please see http://stackoverflow.com/questions/11887762/compare-version-strings

File: restless/dj.py [hunk 2]
```diff
@@ -82,9 +90,15 @@ def urls(cls, name_prefix=None):
...
-        return patterns('',
-            url(r'^$', cls.as_list(), name=cls.build_url_name('list', name_prefix)),
-            url(r'^(?P<pk>\d+)/$', cls.as_detail(), name=cls.build_url_name('detail', name_prefix)),
-        )
+        urls = [url(r'^$', cls.as_list(),
+                    name=cls.build_url_name('list', name_prefix)),
+                url(r'^(?P<pk>\d+)/$', cls.as_detail(),
+                    name=cls.build_url_name('detail', name_prefix))]
+
+        if get_django_version() >= '1.10.0':
```
Comment (seocam) [inline restless/dj.py]: Same here.

File: restless/dj.py (hunk 1 above)
Comment (guneysus) [inline restless/dj.py]: @seocam will be working on  this.

:running_man:

**Comment by guneysus** [comment]
I did not understand the build error:
https://travis-ci.org/toastdriven/restless/jobs/173916340

**Comment by leportella** [comment]
@guneysus maybe just a rebase will work.

I saw that we are not testing 3.3 on the travis file. What do you think?

**Comment by seocam** [comment]
Master should now be compatible with django 1.10

**Pull request closed without merging**